pkg/hive/cell/module.go

```diff
@@ -26,6 +26,13 @@ func Module(id, title string, cells ...Cell) Cell {
 	return &module{id, title, cells}
 }
 
+// ModuleID is the module identifier. Provided in the module's scoped.
+type ModuleID string
+
+// FullModuleID is the fully qualified module identifier, e.g. the
+// concat of nested module ids, e.g. "agent.controlplane.endpoint-manager".
```

Comment:
Wondering if using '/' vs "." could help us here as we could leverage filepath conveniences to assert root module, subpath or expressions to either derive health or others or issue db queries...

Reply:
I don't want to use "/" as then it might make someone think that there's some correspondence to file paths or Go package names. Also "." is the classic module separator and looks nicer.